lib/Windowing/tests/WindowingProvider.spec.js

```diff
@@ -0,0 +1,73 @@
+import {
+  render,
+  fireEvent,
+  waitForElement,
+  queryByTestId
+} from '@testing-library/react';
+import { React } from '../../../tests/setup';
+import { WindowingProvider, InViewList, InViewListItem } from '../..';
+
+jest.mock('debounce', () => fn => fn);
+
+describe('WindowingProvider', () => {
```

Comment:
These tests are great. The fact we can specify `scrollTop: 200` and know exactly what will be rendered and test is so powerful